.gitignore

```diff
@@ -59,3 +59,5 @@ typings/
 
 # next.js build output
 .next
+
+build
```

Comment:
Do you think this should be part of `oz-docs init`?

Reply:
Probably, along with the netlify.toml and the scripts.